diff -Naur belcard-4.4.8/CMakeLists.txt belcard-4.4.8.tpg/CMakeLists.txt
--- belcard-4.4.8/CMakeLists.txt	2020-11-19 09:51:08.052000000 +0000
+++ belcard-4.4.8.tpg/CMakeLists.txt	2020-11-19 09:53:42.200000000 +0000
@@ -139,14 +139,14 @@
   	INSTALL_DESTINATION ${CONFIG_PACKAGE_LOCATION}
 	NO_SET_AND_CHECK_MACRO
 )
-set(ConfigPackageLocation "share/${EXPORT_TARGETS_NAME}/cmake")
+set(CONFIG_PACKAGE_LOCATION share/Belcard/cmake CACHE PATH "CMake configuration file location")
 install(EXPORT ${EXPORT_TARGETS_NAME}Targets
 	FILE ${EXPORT_TARGETS_NAME}Targets.cmake
-	DESTINATION ${ConfigPackageLocation}
+	DESTINATION ${CONFIG_PACKAGE_LOCATION}
 )
 install(FILES
 	"${CMAKE_CURRENT_BINARY_DIR}/${EXPORT_TARGETS_NAME}Config.cmake"
-	DESTINATION ${ConfigPackageLocation}
+	DESTINATION ${CONFIG_PACKAGE_LOCATION}
 )
 
 if (ENABLE_PACKAGE_SOURCE)
