diff -Naur belcard-4.4.8/CMakeLists.txt belcard-4.4.8.tpg/CMakeLists.txt
--- belcard-4.4.8/CMakeLists.txt	2020-06-03 07:24:02.000000000 +0000
+++ belcard-4.4.8.tpg/CMakeLists.txt	2020-11-19 09:51:08.052000000 +0000
@@ -80,6 +80,10 @@
 endif()
 add_definitions("-DBELCARD_EXPORTS")
 
+set(PACKAGES_PRIVATE "belr bctoolbox")
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/belcard.pc.in ${CMAKE_CURRENT_BINARY_DIR}/belcard.pc)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/belcard.pc DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+
 set(STRICT_OPTIONS_CPP )
 set(STRICT_OPTIONS_C )
 set(STRICT_OPTIONS_CXX )
diff -Naur belcard-4.4.8/belcard.pc.in belcard-4.4.8.tpg/belcard.pc.in
--- belcard-4.4.8/belcard.pc.in	2020-06-03 07:24:02.000000000 +0000
+++ belcard-4.4.8.tpg/belcard.pc.in	2020-11-19 09:49:28.837000000 +0000
@@ -5,5 +5,5 @@
 Description: Belcard is Belledonne Communications' vCard 4 parsing library
 Version: @PROJECT_VERSION@
 Libs: -L@CMAKE_INSTALL_FULL_LIBDIR@ -lbelcard
-Libs.private: @LIBS_PRIVATE@
+Requires.private: @LIBS_PRIVATE@
 Cflags: -I@CMAKE_INSTALL_FULL_INCLUDEDIR@
