diff -Naur belcard-4.4.8/CMakeLists.txt belcard-4.4.8.tpg/CMakeLists.txt
--- belcard-4.4.8/CMakeLists.txt	2020-06-03 07:24:02.000000000 +0000
+++ belcard-4.4.8.tpg/CMakeLists.txt	2020-11-19 12:46:34.400000000 +0000
@@ -135,7 +135,7 @@
   	INSTALL_DESTINATION ${CONFIG_PACKAGE_LOCATION}
 	NO_SET_AND_CHECK_MACRO
 )
-set(ConfigPackageLocation "share/${EXPORT_TARGETS_NAME}/cmake")
+set(ConfigPackageLocation "${CMAKE_INSTALL_LIBDIR}/cmake/${EXPORT_TARGETS_NAME}")
 install(EXPORT ${EXPORT_TARGETS_NAME}Targets
 	FILE ${EXPORT_TARGETS_NAME}Targets.cmake
 	DESTINATION ${ConfigPackageLocation}
